readme.adoc

```diff
@@ -505,7 +505,55 @@ https://github.com/aws-samples/aws-microservices-deploy-options/issues/104
 
 === Deployment Pipeline: Fargate with AWS CodePipeline
 
-https://github.com/aws-samples/aws-microservices-deploy-options/issues/82
+### Deployment Pipeline
+This section will explain how to deploy a Fargate task via CodePipeline
+
+. Create a fork of the Github repository that contains the [Amazon ECS Sample App](https://github.com/awslabs/ecs-demo-php-simple-app).  
```

Comment:
That's true, however, it might be useful to show how users can push changes from a cloned repository on their local machine and watch as they move through the pipeline.